lib/import_csv_to_big_query.rb

```diff
@@ -0,0 +1,27 @@
+require 'google/cloud/bigquery'
+require 'rollbar'
+require 'export_tables_to_cloud_storage'
+require 'active_support'
+include ActiveSupport::Inflector
+
+class ImportCSVToBigQuery
+  def load(bigquery: Google::Cloud::Bigquery.new)
+    dataset  = bigquery.dataset ENV['BIG_QUERY_DATASET']
+
+    ExportTablesToCloudStorage::TABLES.each do |table_csv|
+      import_csv_uri  = "gs://tvs_staging/csv_export/#{underscore(table_csv)}.csv"
```

Comment:
You are hardcoding the bucket name here `tvs_staging ` you should take it from the environment variable `CLOUD_STORAGE_BUCKET`

Reply:
Good spot! Will get this fixed up 😄